### PR TITLE
[fr] Replace use of deprecated `fullversion` param

### PR DESCRIPTION
**Pull request by A-Hilaly**
Use `{{% skew currentPatchVersion %}}` shortcode instead.

Fixes https://github.com/kubernetes/website/issues/41290

/cc @sftim @feloy

### CODE REVIEW

File: content/fr/docs/setup/learning-environment/minikube.md [hunk 1]
```diff
@@ -218,10 +218,10 @@ Pour revenir à ce contexte, exécutez la commande suivante: `kubectl config use
 #### Spécifier la version de Kubernetes
 
 Vous pouvez spécifier la version de Kubernetes pour Minikube à utiliser en ajoutant la chaîne `--kubernetes-version` à la commande `minikube start`.
-Par exemple, pour exécuter la version {{< param "fullversion" >}}, procédez comme suit:
+Par exemple, pour exécuter la version {{< skew currentPatchVersion >}}, procédez comme suit:
 
 ```shell
-minikube start --kubernetes-version {{< param "fullversion" >}}
+minikube start --kubernetes-version {{< skew currentPatchVersion >}}
```
Comment (sftim) [inline content/fr/docs/setup/learning-environment/minikube.md:224]: Should this be:
```suggestion
minikube start --kubernetes-version v{{< skew currentPatchVersion >}}
```

**Review by sftim — COMMENTED** [review]
One question

File: content/fr/docs/setup/learning-environment/minikube.md (hunk 1 above)
Comment (A-Hilaly) [reply content/fr/docs/setup/learning-environment/minikube.md]: Fixed

**Review by sftim — COMMENTED** [review]
/lgtm

**Pull request merged**